### ISSUE TITLE
Add a matrix of currently supported bootable containers / links to them

### ISSUE THREAD

**Issue by cdrage** [containers/bootable#6]
Add a list of which containers are available (official as well as third-party?) that can be used with bootc image builder / bootc / the tooling.

For example, adding a matrix that links to documentation for https://github.com/CentOS/centos-bootc/ fedora, etc.

**Comment by stefwalter** [comment]
Good idea. Would you be interested in getting it started ... and then I'd poke multiple people to contribute to it?

For example, it should include Fedora and CentOS Stream images, and Universal Blue images. ... and probably others.

**Comment by imbev** [comment]
If I may,

HeliumOS: `oci.heliumos.org/heliumos/bootc:10`
